src/documentation/ToolkitDocumentation/api.ts

```diff
@@ -17,32 +18,42 @@ const toolkitQuery = `
       }
     }
   }`;
+};
 // No need to add a Sanity client dependency just for this.
-const toolkitQueryUrl =
-  "https://ajwvhvgo.api.sanity.io/v1/data/query/apps?query=" +
-  encodeURIComponent(toolkitQuery);
+const toolkitQueryUrl = (languageId: string): string => {
+  return (
+    "https://ajwvhvgo.api.sanity.io/v1/data/query/apps?query=" +
+    encodeURIComponent(toolkitQuery(languageId))
+  );
+};
 
-const fetchToolkitInternal = async (): Promise<Toolkit> => {
-  const response = await fetch(toolkitQueryUrl);
-  if (response.ok) {
-    const { result } = await response.json();
-    if (!result) {
-      throw new Error("Unexpected response format");
-    }
-    const toolkits = result as Toolkit[];
-    if (toolkits.length !== 1) {
-      throw new Error("Could not find expected toolkit");
+const fetchToolkitExternal = async (languageId: string): Promise<Toolkit> => {
+  let numRequests = 0;
+  const sanitisedLanguageId = languageId.match(/[a-z]+/g)?.join() ?? "";
+
+  const fetchToolkitInternal = async (languageId: string): Promise<Toolkit> => {
+    const response = await fetch(toolkitQueryUrl(languageId));
+    numRequests++;
+    if (response.ok) {
+      const { result } = await response.json();
+      if (!result) {
+        throw new Error("Unexpected response format");
+      }
+      const toolkits = result as Toolkit[];
+      if (toolkits.length !== 1 && numRequests === 1) {
+        return fetchToolkitInternal("en");
+      }
```

Comment:
This is the neatest way I've found to fall back to English without creating an infinite loop. Open to suggestions if there is a better way.